common/src/types/v0/message_bus/volume.rs

```diff
@@ -505,3 +513,25 @@ impl DestroyVolume {
         &self.uuid
     }
 }
+
+/// Replica topology information
+#[derive(Serialize, Deserialize, Default, Debug, Clone, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct ReplicaTopology {
+    /// id of the mayastor instance
+    pub node: Option<NodeId>,
```

Comment:
do these need to be pub?

Reply:
No they don't - I've made them private and created a `new` method.